src/shared_lib/lib_jieqi.cpp

```diff
@@ -129,7 +128,7 @@ struct JieqiMomentQuery {
  * @param jq_idx The index of the Jieqi. Expected to be in the range [0, 24). This is the enum value of `Jieqi`.
  * @returns A `JieqiMomentQuery` struct.
  */
-JieqiMomentQuery query_jieqi(const int32_t year, const int32_t jq_idx) {
+auto query_jieqi(const int32_t year, const int32_t jq_idx) -> JieqiMomentQuery { // NOLINT(bugprone-easily-swappable-parameters)
```

Comment:
Also add a function to convert Jieqi enum to Jieqi index. And also add a function to list all Jieqi enums?

Reply:
Created https://github.com/0xf3cd/celestial-calendar/issues/25. Not fixing in this PR.